lib/modules/apostrophe-pages/lib/api.js

```diff
@@ -369,22 +408,15 @@ module.exports = function(self, options) {
       return setImmediate(callback);
     }
     function permissions(callback) {
-      if (!moved._publish) {
-        return callback(new Error('forbidden'));
-      }
-      // You can always move a page into the trash. You can
-      // also change the order of subpages if you can
-      // edit the subpage you're moving. Otherwise you
-      // must have edit permissions for the new parent page.
-      if ((oldParent._id !== parent._id) && (parent.type !== 'trash') && (!parent._edit)) {
-        return callback(new Error('forbidden'));
-      }
-      return callback(null);
+      return self.apos.callAll('pageMovePermissions', req, moved, {
```

Comment:
Take a look at all `callAll`s and determine if they should be replaced by using improve on the `apostrophe-pages` module.